blosc2/SChunk.py

```diff
@@ -315,6 +315,70 @@ def update_data(self, nchunk, data, copy):
         blosc2_ext._check_access_mode(self.urlpath, self.mode)
         return super(SChunk, self).update_data(nchunk, data, copy)
 
+    def get_slice(self, start=0, stop=None, out=None):
+        """Get a slice from :paramref:`start` to :paramref:`stop`.
+
+        Parameters
+        ----------
+        start: int
+            The nth item where the slice will begin. Default is 0.
+        stop: int
+            The nth item where the slice will end (without including it).
+            Default is until the SChunk end.
```

Comment:
end -> ends

Reply:
Fixed in 20b3d298e5aef2b2731aabf104d51dc0a67dcaca.